src/manifold/src/impl.cpp

```diff
@@ -242,20 +242,17 @@ struct CoplanarEdge {
     const int edgeIdx = thrust::get<2>(inOut);
 
     const Halfedge edge = halfedge[edgeIdx];
-    if (!edge.IsForward()) return;
```

Comment:
This fixed a bug I found - I was not deduplicating property verts correctly before.